app/assets/javascripts/clipboard.js

```diff
@@ -0,0 +1,37 @@
+$(document).on("turbolinks:load", function () {
+  addCopyButton();
+});
+
+function addCopyButton() {
+  new Clipboard('.clipboard-copy');
+
+  $("pre").each(function(index) {
+    var id = "id_" + index;
+    $(this).children("code").attr("id", id);
```

Comment:
you may want to assign $(this) to a variable:
`$this = $(this)`
that way it caches the fetch from the DOM for all three calls.